sonar-ws/src/main/gen-java/org/sonarqube/ws/QualityProfiles.java

```diff
@@ -0,0 +1,2483 @@
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    int getProfilesCount();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    java.util.List<? extends org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfileOrBuilder> 
+        getProfilesOrBuilderList();
+    /**
+     * <code>repeated .sonarqube.ws.rules.WsSearchResponse.QualityProfile profiles = 1;</code>
+     */
+    org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfileOrBuilder getProfilesOrBuilder(
+        int index);
+  }
+  /**
+   * Protobuf type {@code sonarqube.ws.rules.WsSearchResponse}
+   *
+   * <pre>
+   * WS api/qualityprofiles/search
+   * </pre>
+   */
+  public static final class WsSearchResponse extends
+      com.google.protobuf.GeneratedMessage implements
+      // @@protoc_insertion_point(message_implements:sonarqube.ws.rules.WsSearchResponse)
+      WsSearchResponseOrBuilder {
+    // Use WsSearchResponse.newBuilder() to construct.
+    private WsSearchResponse(com.google.protobuf.GeneratedMessage.Builder<?> builder) {
+      super(builder);
+      this.unknownFields = builder.getUnknownFields();
+    }
+    private WsSearchResponse(boolean noInit) { this.unknownFields = com.google.protobuf.UnknownFieldSet.getDefaultInstance(); }
+
+    private static final WsSearchResponse defaultInstance;
+    public static WsSearchResponse getDefaultInstance() {
+      return defaultInstance;
+    }
+
+    public WsSearchResponse getDefaultInstanceForType() {
+      return defaultInstance;
+    }
+
+    private final com.google.protobuf.UnknownFieldSet unknownFields;
+    @java.lang.Override
+    public final com.google.protobuf.UnknownFieldSet
+        getUnknownFields() {
+      return this.unknownFields;
+    }
+    private WsSearchResponse(
+        com.google.protobuf.CodedInputStream input,
+        com.google.protobuf.ExtensionRegistryLite extensionRegistry)
+        throws com.google.protobuf.InvalidProtocolBufferException {
+      initFields();
+      int mutable_bitField0_ = 0;
+      com.google.protobuf.UnknownFieldSet.Builder unknownFields =
+          com.google.protobuf.UnknownFieldSet.newBuilder();
+      try {
+        boolean done = false;
+        while (!done) {
+          int tag = input.readTag();
+          switch (tag) {
+            case 0:
+              done = true;
+              break;
+            default: {
+              if (!parseUnknownField(input, unknownFields,
+                                     extensionRegistry, tag)) {
+                done = true;
+              }
+              break;
+            }
+            case 10: {
+              if (!((mutable_bitField0_ & 0x00000001) == 0x00000001)) {
+                profiles_ = new java.util.ArrayList<org.sonarqube.ws.QualityProfiles.WsSearchResponse.QualityProfile>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 19. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)